tableauserverclient/server/request_factory.py

```diff
@@ -313,6 +312,13 @@ def publish_req_chunked(self, workbook_item, connection_credentials=None):
         parts = {'request_payload': ('', xml_request, 'text/xml')}
         return _add_multipart(parts)
 
+class WorkbookConnection(object):
+    def update_req(self, connection_item):
+        xml_request = ET.Element('tsRequest')
+        connection_element = ET.SubElement(xml_request, 'connection')
+        connection_element.attrib['serverAddress'] = connection_item.server_address.lower()        
```

Comment:
No particular reason - I was just editing in parts that I needed to change. Probably a good idea to add the other ones too.